### PR TITLE
Support asset collections for filtering

### PR DESCRIPTION
**Pull request by kdambekalns**
This provides a way to filter on the pixx.io side by the selected asset collection.
This allows to provide the category (folder) structure in pixx.io on the Neos side.

To support this, a CLI command is added to import (a configurable set of)
categories into Neos as asset collections.

### CODE REVIEW

File: Classes/Command/PixxioCommandController.php [hunk 1]
```diff
@@ -195,4 +224,80 @@ public function updateMetadataCommand(string $assetSource = 'flownative-pixxio',
...
+     * Import pixx.io categories as asset collections
+     *
+     * @param string $assetSourceIdentifier Name of the pixx.io asset source (defaults to "flownative-pixxio")
+     * @param bool $quiet If set, only errors will be displayed.
+     * @param bool $dryRun If set, no changes will be made.
+     * @return void
+     * @throws IllegalObjectTypeException
+     * @throws \Flownative\Pixxio\Exception\ConnectionException
+     */
+    public function importCategoriesAsCollectionsCommand(string $assetSourceIdentifier = 'flownative-pixxio', bool $quiet = true, bool $dryRun = false): void
```
Comment (mficzel) [inline Classes/Command/PixxioCommandController.php:238]: I think $quiet should be false by default.

**Review by mficzel — COMMENTED** [review]
I can confirm the import via cli and the configuration works for the import works as documented.

However i can only see the collections in the Neos AssetSource and not the PixxIO asset source. This may be a problem of the Neos version i tested with. (currently 5.3, update to 7.3 is prepared)

**Comment by kdambekalns** [comment]
> However i can only see the collections in the Neos AssetSource and not the PixxIO asset source. This may be a problem of the Neos version i tested with. (currently 5.3, update to 7.3 is prepared)

Yes, as stated in the README with this PR:

```markdown
   The pixx.io asset source declares itself read-only. Neos does not show asset
   collections in the UI for read-only asset sources. This has been changed for
   Neos 7.3.0 and up with https://github.com/neos/neos-development-collection/pull/3481

   If you want to use this feature with older Neos versions, you can use the PR with
   [cweagans/composer-patches](https://github.com/cweagans/composer-patches#readme)
   or copy the adjusted template into your project and use `Views.yaml` to activate it.
```

File: Classes/Command/PixxioCommandController.php (hunk 1 above)
Comment (kdambekalns) [reply Classes/Command/PixxioCommandController.php:238]: I'll keep it true by default for two reasons:

- consistency with the Canto asset source CLI behaviour
- this is supposed to be run in a cron job, so quiet is the more probably case (IMHO)

**Pull request merged**